source/hook.cpp

```diff
@@ -46,6 +46,8 @@ reshade::hook::status reshade::hook::uninstall()
 	if (!valid())
 		return hook::status::unsupported_function;
 
+	s_export_hook_path.clear();
```

Comment:
I don't understand this change.

Reply:
Removed it. It is just that s_export_hook_path.clear(); makes the game to crash with an invalid hook call, because it tries to hook an empty module path